open-metadata-implementation/adapters/open-connectors/rest-client-connectors/spring-rest-client-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/restclients/spring/SpringRESTClientConnector.java

```diff
@@ -75,7 +75,15 @@ public SpringRESTClientConnector() throws NoSuchAlgorithmException, KeyStoreExce
         /* Ensure that the REST template always uses UTF-8 */
         List<HttpMessageConverter<?>> converters = restTemplate.getMessageConverters();
         converters.removeIf(httpMessageConverter -> httpMessageConverter instanceof StringHttpMessageConverter);
-        converters.add(0, new StringHttpMessageConverter(StandardCharsets.UTF_8));
+        converters.removeIf(httpMessageConverter -> httpMessageConverter instanceof MappingJackson2HttpMessageConverter);
```

Comment:
I am unsure of the implications of this change - as it effects all Spring clients. Please could you detail in the issue and in the comments the need for this change. Does this work if the client is talking to a back level server -if the client is using a newer way to serialise Jackson. I think this change needs wider review - I will include other

Reply:
@immmus Another thought - what is the implications of not have the spring rest client change. Is this required to fix the reported problem? If the Spring client changes are removed from the pr and the problem is still fixed I will merge. If this is the case, you can then submit these changes and justifications separately in another issue.

Reply:
> @immmus Another thought - what is the implications of not have the spring rest client change. Is this required to fix the reported problem? If the Spring client changes are removed from the pr and the problem is still fixed I will merge. If this is the case, you can then submit these changes and justifications separately in another issue.

@davidradl Yes, this is not important change. I can rollback this tomorrow. Comment with confirmation if you want it

Reply:
@immmus not entirely.
It is very good that you removed never thrown exception.
But, the reason why StringHttpMessageConverter is replaced (removed and added) with another instance using UTF-8 encoding instead of default ISO-8859-1 it is explained in the comment of the method.
Shortly, because of using regex as query param. Without it will break the regex. 
That line needs to stay.
Also cannot see the reason why you have removed MappingJackson2HttpMessageConverter instance from the list.

Reply:
> @immmus not entirely.
> It is very good that you removed never thrown exception.
> But, the reason why StringHttpMessageConverter is replaced (removed and added) with another instance using UTF-8 encoding instead of default ISO-8859-1 it is explained in the comment of the method.
> Shortly, because of using regex as query param. Without it will break the regex.
> That line needs to stay.
> Also cannot see the reason why you have removed MappingJackson2HttpMessageConverter instance from the list.

I noticed that a comment was left to make sure the Rest Template always uses UTF-8. But since I knew that MappingJackson2HttpMessageConverter, since the default charset is not set, I decided to set it in accordance with this statement. I wrote about it in my comment -> https://github.com/odpi/egeria/pull/3462#issuecomment-670481314